src/components/designSystem/Icon/Icon.tsx

```diff
@@ -79,53 +74,13 @@ export const Icon = ({
   const SVGIcon = ALL_ICONS[name]
 
   return (
-    <StyledIcon
+    <SVGIcon
       title={`${name}/${size}`}
       data-test={`${name}/${size}`}
-      $size={size}
-      $canClick={!!onClick}
-      className={cx('svg-icon', className, { [`icon-animation--${animation}`]: animation })}
-      $color={mapColor(color)}
-      component={<SVGIcon />}
+      className={tw(iconStyles({ animation, color, size }), className, {
+        'cursor-pointer': !!onClick,
+      })}
       onClick={onClick}
     />
   )
 }
-
-const StyledIcon = styled(({ component, ...props }) => cloneElement(component, props))`
-  width: ${(props: { $size: keyof typeof IconSizeEnum }) => IconSizeEnum[props.$size]};
-  min-width: ${(props: { $size: keyof typeof IconSizeEnum }) => IconSizeEnum[props.$size]};
-  height: ${(props: { $size: keyof typeof IconSizeEnum }) => IconSizeEnum[props.$size]};
-  color: ${(props) => props.$color};
-  cursor: ${({ $canClick }) => ($canClick ? 'pointer' : 'initial')};
-
-  &.icon-animation--${IconAnimationEnum.spin} {
-    animation: spin 1s linear infinite;
-  }
-
-  &.icon-animation--${IconAnimationEnum.pulse} {
-    animation: pulse 1.5s ease-in-out 0.5s infinite;
```

Comment:
It seems that pulse animation from tailwind doesn't use the same easing and delays: https://tailwindcss.com/docs/animation#pulse. Maybe we have to update our config

Reply:
yep, thought no one would notice haha

We don't use this API today so I decided to keep with the default Tailwind config 😬